components/logo/logo.vue

```diff
@@ -0,0 +1,18 @@
+<template>
```

Comment:
This is not a logo, a logo is for a company or stuffs like that. This is either an image or icon I would go with icon

Reply:
Done.